pilot/pkg/serviceregistry/kube/controller/network_test.go

```diff
@@ -257,9 +257,10 @@ func TestAmbientSystemNamespaceNetworkChange(t *testing.T) {
 	testNS := "test"
 	systemNS := "istio-system"
 
+	networksWatcher := meshwatcher.NewFixedNetworksWatcher(nil)
```

Comment:
@As long as none of these tests _write_ to networks, this should be ok, but #54604 has me scared straight when it comes to shared state